app/services/submit_reference.rb

```diff
@@ -18,11 +18,39 @@ def save!
     if @send_emails
       CandidateMailer.reference_received(reference).deliver_later
       RefereeMailer.reference_confirmation_email(application_form, reference).deliver_later
+      notify_provider_users
```

Comment:
We'll need to put this behind a feature flag for now: `new_references_flow_providers`. Just to make sure this email will only be sent once we've turned the new feature on for providers